website/docs/r/waf_byte_match_set.html.markdown

```diff
@@ -50,7 +50,8 @@ The following arguments are supported:
   e.g., `CONTAINS`, `CONTAINS_WORD` or `EXACTLY`.
   See [docs](http://docs.aws.amazon.com/waf/latest/APIReference/API_ByteMatchTuple.html#WAF-Type-ByteMatchTuple-PositionalConstraint)
   for all supported values.
-* `target_string` - (Optional) The value that you want to search forE.g., `HEADER`, `METHOD` or `BODY`.
+* `target_string` - (Optional) The value that you want to search for.
+  e.g., `badrefer1`.
```

Comment:
Combined these two into one line and tried to clarify the wording a bit. Does this change still appropriately represent your suggestion?

```suggestion
* `target_string` - (Optional) The value that you want to search for within the field specified by `field_to_match`, e.g., `badrefer1`.
```